docs/tutorials/bigip-ha.rst

```diff
@@ -0,0 +1,130 @@
+.. index::
+   single: BIG-IP Controller
+   single: BIG-IP high availability
+   single: Cloud Foundry
+   single: Kubernetes
+   single: OpenShift
+   single: Marathon
+
+.. _manage BIG-IP HA:
+
+How to manage BIG-IP HA pairs
+=============================
+
+The F5 Container Connectors provide platform-native integrations for BIG-IP devices from PaaS providers like Cloud Foundry, Kubernetes, Mesos, & OpenShift. The BIG-IP Controllers for these platforms translate native commands to F5 Python SDK/iControl REST calls. [#cccl]_
```

Comment:
Is the CCCL reference necessary here? Do we not mention it in other places?

Reply:
This is the only place I've mentioned it in the docs so far. That's not to say it shouldn't be mentioned somewhere else...

Reply:
It just seems out of place to me here. Would be better in the introduction to the CCs. I don't feel it's necessary, so I'd recommend removing it.